api/user/index.ts

```diff
@@ -5,7 +5,7 @@ import { getUserSessionTimeout } from './userTimeout'
 
 export async function getUserDetails(req, res: Response, next: NextFunction) {
 
-  const { roles } = req.session.user
+  const { roles } = req.session.passport.user.userinfo
```

Comment:
this looks awkard any reason to wrap with passport

Reply:
the library we've pushed uses passport under the hood; if we then stored user back onto ```req.session``` there would be a lot of data that would have to be transferred back/forth to redis.